### PR TITLE
Visualize sum of channels in widgets

### PR DESCRIPTION
**Pull request by nontas**
Until now, when trying to visualize the sum of the channels of an image in `visualize_images`, we were using the `glyph` function with `vectors_block_size=1`. 

This PR changes this and actually sums the image's channels. This is done by defining the `sum_channels()` method in `menpo.visualize.image`. The result is visually better and is also faster to compute.

### CODE REVIEW

**Comment by nontas** [comment]
This is now ready. It can get in as soon as the tests pass!

File: menpo/visualize/image.py [hunk 1]
```diff
@@ -4,6 +4,27 @@
...
+    pixels : `ndarray`
+        The input image's pixels.
+    channels : `int`
+        Defines the channels to be summed.
+    """
+    # if channels is None, then all channels are used
+    if channels is None:
+        channels = range(pixels.shape[0])
+    # compute sum image
+    sum_image = np.sum(pixels[channels], axis=0)
```
Comment (patricksnape) [inline menpo/visualize/image.py]: In the default case where no channels are chosen, this is twice as slow. Perhaps you should split the sum in to two between the two cases?

File: menpo/visualize/image.py [hunk 2]
```diff
@@ -4,6 +4,27 @@
...
+    # return as c-contiguous
+    return np.ascontiguousarray(sum_image[None, ...])  # add a channel axis
```
Comment (patricksnape) [inline menpo/visualize/image.py]: ``s.reshape((1,) + s.shape)`` does the same thing without requiring the copy. So you can remove the ``np.ascontiguousarray`` as well.

**Comment by patricksnape** [comment]
Whoops, this also adds all the doc changes I did... but they aren't that important so we can just use this to pull them in as well

**Comment by jabooth** [comment]
@nontas is this ready?

**Comment by nontas** [comment]
Yes! The methods `glyph` and `sum_channels` are now moved to `menpo.feature.visualize` and added to the docs. You can get it in if you're happy!

**Pull request merged**